Added the new antctl check command documentation (#6373)
antrea-io/antrea: docs/antctl.md

antctl get featuregates
```

### Performing checks to facilitate installation process

Antrea provides a utility command `antctl check` designed to perform checks that verify whether the Kubernetes cluster is correctly configured for installing Antrea,
and also to confirm that Antrea has been installed correctly.

#### Pre-installation checks

Before installing Antrea, it's crucial to ensure that the Kubernetes cluster is configured properly. This can prevent potential issues that might arise during the
installation of Antrea. To perform these pre-installation checks, Simply run the command as follows:

> 

Reviewer: Please wrap lines into 80~100 chars.

```bash
antctl check cluster
```

#### Post-installation checks

Once Antrea is installed, it's important to verify that it is functioning correctly within your cluster.

> To perform, Simply run the command:

Reviewer: ```suggestion
To perform post-installation checks, Simply run the command as follows:
```

```bash
antctl check installation
```

> Incase Antrea is installed in a custom namespace, You can specify the namespace by adding the flag:

Reviewer: ```suggestion
In case Antrea is installed in a custom namespace, You can specify the namespace by adding the flag:
```

```bash
antctl check installation --namespace [namespace]
```

Replace `[namespace]` with the actual name of the namespace.

Run the following command to discover more utility flags:

```bash

> antctl check installation --help

Reviewer: I didn't see you add a similar section for `antctl check cluster`, no help for `antctl check cluster`?

Author: Added now, I considered lack of flags a reason to not add that. 
Thanks :)

antctl get featuregates
```

### Performing checks to facilitate installation process

Antrea provides a utility command `antctl check` designed to perform checks that
verify whether the Kubernetes cluster is correctly configured for installing
Antrea, and also to confirm that Antrea has been installed correctly.

#### Pre-installation checks

Before installing Antrea, it's crucial to ensure that the Kubernetes cluster is
configured properly. This can prevent potential issues that might arise during the installation

> of Antrea. To perform these pre-installation checks, Simply run the command as follows:

Reviewer 2: The lines are not wrapped consistently, please wrap them at 80 chars per line.

It applies to other contents too.

run the command as follows:

```bash
antctl check cluster
```

Run the following command to discover more options:

```bash
antctl check cluster --help
```

#### Post-installation checks

Once Antrea is installed, it's important to verify that it is functioning

> correctly within your cluster. To perform post-installation checks, Simply run

Reviewer 3: ```suggestion
correctly within your cluster. To perform post-installation checks, simply run
```

```bash
antctl check installation
```

In case Antrea is installed in a custom namespace, You
can specify the namespace by adding the flag:

```bash

> antctl check installation --namespace [namespace]

Reviewer 2: ```suggestion
antctl check installation --namespace [NAMESPACE]
```
use upper case for placeholder.

```bash
antctl check installation --namespace [NAMESPACE]
```

> Run the following command to discover more utility flags:

Reviewer 3: ```suggestion
Run the following command to discover more options:
```

(for consistency with the prior section)

> Replace `[namespace]` with the actual name of the namespace.

Reviewer 2: This may be unnecessary.
